### ISSUE TITLE
drpcli --endpoint parameter doesn't seem to work

### ISSUE THREAD

**Issue by CoRfr** [digitalrebar/provision#76]
As I tried to use `drpcli` remotely, I tried to use the `--endpoint` parameter to point to my remote host:
```
$ drpcli --endpoint https://10.1.60.1:8092 --token <obfuscated> users list
Error: CLIENT_ERROR: Get https://127.0.0.1:8092/api/v3/users: dial tcp 127.0.0.1:8092: connect: connection refused
```

This is with latest release `drpcli` v4.0.21 from https://rebar-catalog.s3-us-west-2.amazonaws.com/drp/v4.0.21.zip

**Comment by galthaus** [comment]
Do you have RS_ENDPOINT set?

**Comment by CoRfr** [comment]
`RS_ENDPOINT` was not set.

If I do set `RS_ENDPOINT` it works.

**Comment by galthaus** [comment]
Okay - I think we have a bug we fixed that will be resolved in v4.0.22 or v4.1.0 (I think the later is the next release).  Specifically, this #73 .

**Comment by CoRfr** [comment]
Sounds good, thanks @galthaus
